frontend/src/components/contactsForm.tsx

```diff
@@ -0,0 +1,310 @@
+                autoComplete="on"
+                required
+                value={notes}
+                onChange={handleNotesChange}
+              ></input>
+
+              <label>
+                Relationship with Person
+                <input
+                  type="text"
+                  value={relationshipType}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    if (/^[A-Za-z\s]*$/.test(value)) {
+                      setRelationshipType(value)
+                    }
+                  }}
+                />
+              </label>
+              <button type="button" onClick={addRelationship}>Add</button>
+              {relationships.map((type, index) => (
+                <div key={index}>
+                  <input
+                    type="text"
+                    value={type.relationshipType}
+                    onChange={(event) => { handleRelationshipChange(index, 'relationshipType', event.target.value) }
+                    }
+                  />
+                  <button type= "button" onClick={() => { handleRemoveRelationshipType(index) }}>
+                    x
+                  </button>
+                </div>
+              ))}
+
+              <label>
+                Date:
+                <input
+                  type="text"
+                  value={eventDate}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    setEventDate(value)
+                  }}
+                />
+              </label>
+
+              <label>
+                Event:
+                <input
+                  type="text"
+                  value={eventType}
+                  onChange={(event) => {
+                    const value = event.target.value
+                    if (/^[A-Za-z\s]*$/.test(value)) {
+                      setEventType(value)
+                    }
+                  }}
+                />
+              </label>
+              <button type='button' onClick={addImportantEvent}>Add</button>
+              {importantDates.map((date, index) => (
+                <div key={index}>
+                  <input
+                    type="text"
+                    value={date.date}
+                    onChange={(event) => { handleImportantEventChange(index, 'date', event.target.value) }
+                    }
+                  />
+                  <input
+                    type="text"
+                    value={date.eventType}
+                    onChange={(event) => { handleImportantEventChange(index, 'eventType', event.target.value) }
+                    }
+                  />
+                  <button type="button" onClick={() => { handleRemoveImportantEvent(index) }}>
+                    x
+                  </button>
+                </div>
+              ))}
+            </div>
+            <div>
+              <button className="add_contact_button" type="submit">
```

Comment:
i think we're using camelCase in css already so let's keep it consistent one way or the other